_posts/2018-02-07-cockpit-161.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Cockpit 161
+author: pitti
+date: 2018-02-07 10:00
+tags: cockpit linux
+slug: cockpit-161
+category: release
+summary: Cockpit with Virtual Machines, LVM, and Hardware Information improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 161.
+
+### New VMs can be created on Machines page
+
+The Machines page now has a "Create New VM" button for installing a new libvirt
+virtual machine from a local or remote ISO image. It allows the user to set
+some basic properties such as memory and storage sizes.
+
+![Machines Create new VM](/images/machines-create-iso.png)
+
+Thanks to Dominik Perpeet and suomiy for this feature!
+
+### VMs running in Kubernetes can now be deleted
+
+[Cockpit 160](http://cockpit-project.org/blog/cockpit-160.html) added an
+initial overview of kubevirt Virtual Machines. This page now got a first action
+to delete a running VM.
```

Comment:
Is this really only VMs that are running? Or also stopped VMs?

Reply:
In current kubevirt there is no concept of a "stopped" VM, as they are ephemeral (much like a pod). So "running" and "not existing" are the only two states. AFAIK the kubevirt devs want to introduce a concept of "offline VMs" at some point, but it doesn't exist right now.